src/components/bottomSheet/bottomSheetStyle.ts

```diff
@@ -0,0 +1,65 @@
+import { css } from '@emotion/react';
+
+function bottomSheetStyle() {
+  const BackgroundOverlay = (isVisible: boolean) => css`
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100vw;
+    height: 100vh;
+    z-index: 1000;
+    background: rgba(0, 0, 0, 0.5);
+     transition: opacity 0.5s ease;
+     opacity: ${isVisible ? 1 : 0};
+  `;
+
+  const ModalStyle = (isVisible: boolean) => css`
+    position: fixed;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: fit-content;
+    box-sizing: border-box;
+    background: white;
```

Comment:
p1; 범용적으로 적용될 수 있는 색상의 경우(ex. 루트 컨테이너 내부의 카드, 모달의 배경색 등 색상 하나를 여러 곳에 사용하는 경우) useTheme를 통해 불러온 후 사용하면 좋을 것 같습니다. 현재는 아래와 같이 변경하는 것이 좋겠네요.

아래 suggestion은 함수의 상단에 `const theme = useTheme();`로 테마 객체를 불러온 상황을 가정합니다.

```suggestion
    background: ${theme.colors.background.main};
```